dephell/actions/_contributing.py

```diff
@@ -0,0 +1,95 @@
+from typing import Any, Dict, Optional
+
+CONTRIBUTING = """
```

Comment:
That's a big bunch of code. What do you think about making it Jinja2-based and moving it into templates?